cmd/scollector/collectors/snmp_ifaces.go

```diff
@@ -103,9 +103,11 @@ func c_snmp_ifaces(community, host string) (opentsdb.MultiDataPoint, error) {
 		ifTypes[k] = val
 	}
 	for k, v := range ifAliasesRaw {
+		// Replace spaces with underscores for readability, then clean bad characters
+		newAlias := strings.Replace(fmt.Sprintf("%s", v), " ", "_", -1)
```

Comment:
Were these aliases ever being used in tags before? If so, I would be concerned we are changing tagsets currently in use. If not, this looks pretty good to me.